WalletWasabi.Gui/Controls/WalletExplorer/TransactionViewModel.cs

```diff
@@ -55,7 +56,7 @@ public TransactionViewModel(TransactionInfo model)
 				});
 		}
 
-		private TransactionInfo Model { get; }
+		private TransactionDetailsViewModel Model { get; }
```

Comment:
a property called Model cannot represent a ViewModel.... this should be renamed Details or something...

model is to represent the model that the VM wraps in MVVM

Reply:
ACK.